serverless.yml

```diff
@@ -44,6 +49,11 @@ functions:
           method: get
           cors: true
 
+  streamFunction:
+    handler: scans.worker
+    events:
+      - stream: arn:aws:dynamodb:${opt:region, self:provider.region}:*:table/${self:provider.environment.DYNAMODB_TABLE}/stream/*
```

Comment:
I'm currently stuck on an error with this stream definition...

  An error occurred: StreamFunctionEventSourceMappingDynamodbSshobservatorydev - 1 validation error detected: Value 'arn:aws:dynamodb:us-west-2:*:table/ssh-observatory-dev/stream/*' at 'eventSourceArn' failed to satisfy constraint: Member must satisfy regular expression pattern: arn:(aws[a-zA-Z0-9-]*):([a-zA-Z0-9\-])+:([a-z]{2}((-gov)|(-iso(b?)))?-[a-z]+-\d{1})?:(\d{12})?:(.*) (Service: AWSLambda; Status Code: 400; Error Code: ValidationException; Request ID: 638fe543-e914-4285-b6af-89c6f77946eb).

Reply:
My thought was because I failed to include the stream specification on the table, which I just added.

Reply:
The actual value (when I set it manually) is this: arn:aws:dynamodb:us-west-2:163402512545:table/ssh-observatory-dev/stream/2019-10-23T15:34:12.310

Reply:
it does work when set manually, just need to troubleshoot why the tokenized version isn't picking it up.